options.ts

```diff
@@ -50,7 +50,6 @@ export class OptionsBase {
 		return {
 			all: { type: OptionType.Boolean },
 			json: { type: OptionType.Boolean },
-			watch: { type: OptionType.Boolean },
```

Comment:
Personally I think it's better to return `stream.Readable` from `getFile` method and allow the consumers to decide if the output should be in a file or on stdout. This will allow calling the method from the code and using the stream directly. Consider the case where you need to check the content of a file on device. The current implementation will download the file, so you'll have to read it again in the code. In case you return a Readable stream, you can check the content directly. 
The command (`tns device get-file`) can check the passed arguments and decide whether the content should be shown on the terminal or saved in a file.

However this is just an opinion and I do not insist to implement it this way.